File: app/src/main/java/de/ka/chappted/auth/AuthService.java
package de.ka.chappted.auth;

import android.app.Service;
import android.content.Intent;
import android.os.IBinder;

/**
 * A service to authenticate users using the {@link Authenticator Authenticator} class.
 * <p>
 * Created by Thomas Hofmann on 08.12.17.
 */
public class AuthService extends Service {
    @Override
    public IBinder onBind(Intent intent) {

        Authenticator authenticator = new Authenticator(this);
        return authenticator.getIBinder();
    }

}

File: chapptedapi/src/main/java/de/ka/chapptedapi/auth/AuthService.kt
package de.ka.chapptedapi.auth

import android.app.Service
import android.content.Intent
import android.os.IBinder

/**
 * A service to authenticate users using the [Authenticator] class.
 *
 * Created by Thomas Hofmann on 08.12.17.
 */
class AuthService : Service() {
    override fun onBind(intent: Intent): IBinder? {

        val authenticator = Authenticator(this)
        return authenticator.iBinder
    }

}

